### PR TITLE
update release note for external authz

### PR DESCRIPTION
**Pull request by yangminzhu**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [x] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/news/releases/1.12.x/announcing-1.12/change-notes/index.md [hunk 1]
```diff
@@ -74,12 +74,15 @@ aliases:
...
   ([Pull Request #33914](https://github.com/istio/istio/pull/33914))
 
-- **Added** support for JWT claim based routing on ingress gateways.
+- **Added** support for [JWT claim based routing](/docs/tasks/security/authentication/jwt-route) on ingress gateways.
   ([Pull Request #35762](https://github.com/istio/istio/pull/35762))
 
 - **Fixed** the `EnvoyExternalAuthorizationHttpProvider` to match HTTP headers in a case-insensitive way.
   ([Issue #35220](https://github.com/istio/istio/issues/35220))
 
+- **Graduated** the [external authorization](/docs/tasks/security/authorization/authz-custom) from experimental to alpha stage.
```
Comment (ericvn) [inline content/en/news/releases/1.12.x/announcing-1.12/change-notes/index.md:83]: Graduated is not a valid action: https://github.com/istio/istio/tree/master/releasenotes#release-notes. Promoted would be appropriate.

File: content/en/docs/tasks/security/authorization/authz-custom/index.md [hunk 2]
```diff
@@ -7,11 +7,11 @@ owner: istio/wg-security-maintainers
 test: yes
 ---
 
-This task shows you how to set up an Istio authorization policy using a new experimental value for the [action field](/docs/reference/config/security/authorization-policy/#AuthorizationPolicy-Action), `CUSTOM`,
+This task shows you how to set up an Istio authorization policy using a new value for the [action field](/docs/reference/config/security/authorization-policy/#AuthorizationPolicy-Action), `CUSTOM`,
 to delegate the access control to an external authorization system. This can be used to integrate with [OPA authorization](https://www.openpolicyagent.org/docs/latest/envoy-introduction/),
 [`oauth2-proxy`](https://github.com/oauth2-proxy/oauth2-proxy), your own custom external authorization server and more.
 
-{{< boilerplate experimental-feature-warning >}}
+This feature is currently in alpha stage.
```
Comment (ericvn) [inline content/en/docs/tasks/security/authorization/authz-custom/index.md:14]: I only found two other places (maybe there are more) where we mention a feature as Alpha, and we have that in a warning. I'm not sure if we just don't label all Alpha features or if only a few are documented and those have warnings.

**Review by ericvn — CHANGES_REQUESTED** [review]

File: content/en/news/releases/1.12.x/announcing-1.12/change-notes/index.md (hunk 1 above)
Comment (yangminzhu) [reply content/en/news/releases/1.12.x/announcing-1.12/change-notes/index.md]: fixed, thanks for the link.

File: content/en/docs/tasks/security/authorization/authz-custom/index.md (hunk 2 above)
Comment (yangminzhu) [reply content/en/docs/tasks/security/authorization/authz-custom/index.md]: removed the sentence, I guess we don't need to call out the alpha stage unless it's really needed (I'm fine either way but I don't feel there is much warning needed for this feature).

File: content/en/news/releases/1.12.x/announcing-1.12/change-notes/index.md [hunk 3]
```diff
@@ -74,12 +74,15 @@ aliases:
...
   ([Pull Request #33914](https://github.com/istio/istio/pull/33914))
 
-- **Added** support for JWT claim based routing on ingress gateways.
+- **Added** support for [JWT claim based routing](/docs/tasks/security/authentication/jwt-route) on ingress gateways.
   ([Pull Request #35762](https://github.com/istio/istio/pull/35762))
 
 - **Fixed** the `EnvoyExternalAuthorizationHttpProvider` to match HTTP headers in a case-insensitive way.
   ([Issue #35220](https://github.com/istio/istio/issues/35220))
 
+- **Promoted** the [external authorization](/docs/tasks/security/authorization/authz-custom) from experimental to alpha stage.
```
Comment (ericvn) [inline content/en/news/releases/1.12.x/announcing-1.12/change-notes/index.md:83]:
```suggestion
- **Promoted** the [external authorization](/docs/tasks/security/authorization/authz-custom) feature from Experimental to Alpha.
```

**Review by ericvn — APPROVED** [review]

**Pull request merged**